app/main.py

```diff
@@ -2,0 +1,27 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command_listed = command.split(" ")
+    source_file = command_listed[1]
+
+    if command_listed[0] != "mv" or len(command_listed) != 3:
+        return
+
+    file_path, file_name = os.path.split(command_listed[2])
```

Comment:
The use of `os.path.split` is correct here, but ensure that `file_path` is handled correctly when the destination is a directory. If the destination ends with a '/', it should be treated as a directory, and the source file name should be appended to it.